lms/static/js/student_account/views/LoginView.js

```diff
@@ -113,6 +113,15 @@
                 this.setErrors();
                 this.element.hide( this.$resetSuccess );
 
+                /* Labster Changes
+                 * If there was login error, set title message to suggest
+                 * student login to UK region server
+                 */
+                var urlCheck = new RegExp("([a-zA-Z0-9]+://)?([a-zA-Z0-9_]+:[a-zA-Z0-9_]+@)?([a-zA-Z0-9.-]+\\.[A-Za-z]{2,4})(:[0-9]+)?([^ ])+");
+                if (msg.toString().match(urlCheck)){
+                    this.$errors.find('.message-title').text('We could not sign you in. Wrong Region?');
```

Comment:
why do we need those changes? Are they so important?

Reply:
Yes its quite important when we want to change the title of error message
The code function is checking if message error contains an url, 
if contain url it means the message error referring to region error message and will change the title of message to 'We could not sign you in. Wrong region ?'